videos.md

```diff
@@ -1,43 +1,42 @@
+  - title: "Hacker101's Picks"
+    links:
+      - text: "A Starter's Guide to Pentesting with OWASP"
+        url: /sessions/pentest_owasp
+      - text: "JavaScript for Hackers"
+        url: /sessions/javascript_for_hackers
+      - text: "Server-Side Request Forgery"
+        url: /sessions/ssrf
+      - text: "Mobile Hacking Crash Course"
+        url: /sessions/mobile_crash_course
 ---
 
-<div class="container-fluid">
-  <div class="position-relative overflow-hidden row mb-5 text-center bg-light text-dark" style="border: 1px solid black">
-    <div class="col">
-      <h1 class="display-5 font-weight-normal">Video Lessons</h1>
-      <p class="lead font-weight-normal">These videos will help you learn a diverse set of topics. If you want to see specifics of what a session covers, simply click its title.</p>
-      <a class="btn btn-success mb-2" href="https://www.hacker101.com/start-here">Not sure where to start? Click here!</a>
-    </div>
-    <div class="product-device box-shadow d-none d-md-block"></div>
-    <div class="product-device product-device-2 box-shadow d-none d-md-block"></div>
-  </div>
-</div>
-<div class="container-fluid pt-3">
-  <div class="row">
-    <div class="col-md-7 mb-4 pb-2 text-center text-white">
-      <div class="mb-0">
-        <h2 class="display-5"><a href="sessions/javascript_for_hackers">Featured Video</a></h2>
-      </div>
-      <iframe id="ytplayer" type="text/html" width="100%" height="400" src="https://www.youtube-nocookie.com/embed/FTeE3OrTNoA?rel=0&autoplay=0&origin={{ site.url }}" frameborder="0"></iframe>
-    </div>
-    <div class="col-md-4 mb-4 pb-2 border-left border-success">
-      <h2 class="display-5">Learning Tracks</h2>
-      <ul>
-        <li><a href="playlists/newcomers">Hacker101 for Newcomers</a></li>
-        <li><a href="playlists/burp_suite">Burp Suite</a></li>
-        <li><a href="playlists/pentesting_series">Pentest Series</a></li>
-        <li><a href="playlists/web_hacking">Web Hacking</a></li>
-        <li><a href="playlists/mobile_hacking">Mobile Hacking</a><span class="badge badge-pill badge-danger">Updated!</span></li>
-        <li><a href="playlists/hacktivitycon">H@cktivitycon</a></li>
-        <li><a href="playlists/cryptography">Cryptography</a></li>
-      </ul>
-      <h2> Hacker101's Picks </h2>
-      <li><a href="/sessions/pentest_owasp">A Starters Guide to Pentesting with OWASP</a></li>
-      <li><a href="/sessions/javascript_for_hackers">JavaScript for Hackers</a></li>
-      <li><a href="/sessions/ssrf"> Server-Side Request Forgery</a></li>
-      <li><a href="/sessions/mobile_crash_course">Mobile Hacking Crash Course</a></li>
-    </div>
-  </div>
-</div>
+These videos will help you learn a diverse set of topics. If you want to see specifics of what a session covers, simply click its title.
+
+<a class="btn btn-primary" href="/start-here">
+  Not sure where to start? Click here!
+</a>
+
+## [Featured Video](/sessions/javascript_for_hackers)
```

Comment:
Is there a way to do something like `video_src` with this as well, or not because this is a special page?

Reply:
Definitely can! I can just have the featured session linked differently.